app/src/test/java/com/example/concordia_campus_guide/ModelTests/TestUtils.java

```diff
@@ -1,17 +1,23 @@
 package com.example.concordia_campus_guide.ModelTests;
```

Comment:
Can you make this class specific to routes or something by changing the name of this class? Or else all test utils will end up here and that will make this class extremely cluttered